configs.go

```diff
@@ -1277,3 +1277,23 @@ func (config GetStickerSetConfig) values() (url.Values, error) {
 	v.Add("name", config.Name)
 	return v, nil
 }
+
+// DiceConfig contains information about a sendDice request.
+// Chattable interface
```

Comment:
You can remove the second line of this comment. While it may be useful, it's inconsistent with every other config type not mentioning it.